membership/src/main/java/net/corda/v5/membership/EndpointInfo.java

```diff
@@ -0,0 +1,40 @@
+package net.corda.v5.membership;
+
+import net.corda.v5.base.annotations.CordaSerializable;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * <p>Information about a virtual node's endpoint (e.g. a virtual node's peer-to-peer gateway endpoint).</p>
+ *
+ * <p>Example usages:</p>
+ *
+ * <p>Java:</p>
+ * <pre>{@code
+ * MemberInfo memberInfo = memberLookup.myInfo();
+ * List<EndpointInfo> endpoints = memberInfo.getMemberProvidedContext().parseList("corda.endpoints", EndpointInfo.class);
+ * String url = endpoints.get(0).getUrl();
+ * int protocolVersion = endpoints.get(0).getProtocolVersion();
+ * }</pre>
+ *
+ * <p>Kotlin:</p>
+ * <pre>{@code
+ * val memberInfo = memberLookup.myInfo()
+ * val endpoints = memberInfo.memberProvidedContext.parseList("corda.endpoints", EndpointInfo::class.java)
+ * val url = endpoints.first().url
+ * val protocolVersion = endpoints.first().protocolVersion
+ * }</pre>>
+ */
```

Comment:
The `:base` and `:application` modules are using
```
<ul>
<li>Java:<pre>{@code ...}</pre></li>
<li>Kotlin:<pre>{@code ...}</pre></li>
</ul>
```
which makes for slightly nicer mark-up. And I think we should be consistent too.

This comment applies to _all_ Javadocs here.